api/src/test/java/org/pmiops/workbench/rdr/RdrExportServiceImplTest.java

```diff
@@ -236,6 +248,28 @@ public void exportWorkspace_DeletedWorkspace() throws ApiException {
     verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
   }
 
+  /**
+   * Rdr should send excludeFromDirectory as true if workspace creator does not have verified
+   * institution
+   *
+   * @throws ApiException
+   */
+  @Test
+  public void exportWorkspace_CreatorWithVerifiedInstitution() throws ApiException {
+    mockWorkspace.setCreator(dbUserWithoutVerifiedInstitution);
+    List<Long> workspaceID = new ArrayList<>();
+    workspaceID.add(1l);
+    rdrExportService.exportWorkspaces(workspaceID);
+    verify(mockWorkspaceService)
+        .getFirecloudUserRoles(
+            mockWorkspace.getWorkspaceNamespace(), mockWorkspace.getFirecloudName());
+    verify(rdrExportDao, times(1)).save(anyList());
+
+    RdrWorkspace rdrWorkspace = toDefaultRdrWorkspace(mockWorkspace);
+    rdrWorkspace.setExcludeFromPublicDirectory(true);
+    verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
```

Comment:
This confused me because the `verify` was so far behind the call that should kick off the verified call.

I'd move the definition of the rdrWorkspace above like 
```
    mockWorkspace.setCreator(dbUserWithoutVerifiedInstitution);
    List<Long> workspaceID = new ArrayList<>();
    workspaceID.add(1l);
    RdrWorkspace rdrWorkspace = toDefaultRdrWorkspace(mockWorkspace);
    rdrWorkspace.setExcludeFromPublicDirectory(true);

    rdrExportService.exportWorkspaces(workspaceID);
    verify(mockWorkspaceService)
        .getFirecloudUserRoles(
            mockWorkspace.getWorkspaceNamespace(), mockWorkspace.getFirecloudName());
    verify(rdrExportDao).save(anyList());
    verify(mockRdrApi).exportWorkspaces(Arrays.asList(rdrWorkspace));
```
and possibly declare your array above too.